exporters/otlp/README.md

```diff
@@ -74,10 +87,10 @@ Sets the number of Goroutines to use when processing telemetry.
 Disables client transport security for the exporter's gRPC connection just like [`grpc.WithInsecure()`](https://pkg.go.dev/google.golang.org/grpc#WithInsecure) does.
 By default, client security is required unless `WithInsecure` is used.
 
-### `WithAddress(addr string)`
+### `WithEndpoint(endpoint string)`
 
-Sets the address that the exporter will connect to the collector on.
-The default address the exporter connects to is `localhost:55680`.
+Sets the endpoint that the exporter will connect to the collector on.
+By default the exporter connects to `localhost:55680`.
```

Comment:
```suggestion
By default, the exporter connects to `localhost:4317`.
```
The default collector port is `4317` now.

https://github.com/open-telemetry/opentelemetry-go/blob/38efc875cf1c9e9ae72d3021b0a813ed7fa3ce00/exporters/otlp/options.go#L24